pkg/services/account/account_mock_test.go

```diff
@@ -0,0 +1,88 @@
+package account
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestNewMockAccountService(t *testing.T) {
+	g := NewWithT(t)
+
+	accountService := NewMockAccountService()
+	_, isExpectedType := accountService.(*mock)
+	g.Expect(isExpectedType).To(BeTrue())
+}
+
+func Test_mock_SearchOrganizations(t *testing.T) {
+	type args struct {
+		filter string
+	}
+	tests := []struct {
+		name    string
+		a       mock
+		args    args
+		want    *OrganizationList
+		wantErr bool
+	}{
+		{
+			name: "should return a list of mock Organizations",
+			a:    mock{},
+			args: args{
+				filter: "",
+			},
+			want:    buildMockOrganizationList(10),
+			wantErr: false,
+		},
+	}
+	g := NewWithT(t)
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			a := mock{}
+			got, err := a.SearchOrganizations(tt.args.filter)
+			g.Expect(err != nil).To(Equal(tt.wantErr))
+			g.Expect(got).To(Equal(tt.want))
+
```

Comment:
We are just testing the mock. Is this what we are trying to achieve? What is the use in unit testing the mock?